pkg/systemd/logs.scss

```diff
@@ -70,6 +70,7 @@
 }
 
 #journal .filter select,
```

Comment:
If you keep the select → ul, please clean up the select selector here.

Reply:
There are other `select` that this applies to

Reply:
Ack, thanks.